src/services/balancer/api/entities/pools/query.ts

```diff
@@ -0,0 +1,119 @@
+import { GraphQLArgs } from '@balancer-labs/sdk';
```

Comment:
Should the single pool and mutiple pool queries not have a single source of truth so we know that we only have to update the query in one place to ensure pool data is consistent everywhere?